MoltenVK/MoltenVK/GPUObjects/MVKInstance.mm

```diff
@@ -707,6 +707,8 @@
 	// Device extension functions:
 	ADD_DVC_EXT_ENTRY_POINT(vkCmdPushDescriptorSetKHR, KHR_PUSH_DESCRIPTOR);
 	ADD_DVC_EXT2_ENTRY_POINT(vkCmdPushDescriptorSetWithTemplateKHR, KHR_PUSH_DESCRIPTOR, KHR_DESCRIPTOR_UPDATE_TEMPLATE);
+	ADD_DVC_EXT_ENTRY_POINT(vkMapMemory2KHR, KHR_MAP_MEMORY_2);
+	ADD_DVC_EXT_ENTRY_POINT(vkUnmapMemory2KHR, KHR_MAP_MEMORY_2);
```

Comment:
Niggly point, but can you move these up to the top, as I suggested? 

It is intended to be sorted and grouped by:
1. `KHR` / `EXT` / `VENDOR` groupings
2. Alpha by extension name...so `KHR_MAP...` ahead of `KHR_PUSH...`.